src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGetServices.cs

```diff
@@ -8,14 +8,14 @@ namespace NuGet.VisualStudio.Internal.Contracts
 {
     public static class NuGetServices
     {
-        public static string NuGetSolutionServiceName = "NuGetSolutionService";
-        public static string NuGetSolutionServiceVersion = "1.0.0";
+        internal static string NuGetSolutionServiceName = "NuGetSolutionService";
+        internal static string NuGetSolutionServiceVersion = "1.0.0";
```

Comment:
```suggestion
        internal static const string NuGetSolutionServiceVersion = "1.0.0";
```

Reply:
That doesn't make sense, a const object is always static, it's part of the C# language spec.

Reply:
I think his suggestion was meant to be `const` without `static`... since this string shouldn't changeable.

Reply:
That would violate https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1052?view=vs-2019.

Reply:
Actually, maybe it doesn't...idk I'd have to install the analyzers but kartheek did provide that feedback in some of my previous PRs.

Reply:
I've changed it.. @kartheekp-ms's baselining will catch it if it needs changing.